toolkits/slack/evals/eval_slack_chat.py

```diff
@@ -0,0 +1,625 @@
+            list_public_channels_metadata,
+        ),
+        (
+            "List private channels",
+            "List all private channels",
+            list_private_channels_metadata,
+        ),
+        (
+            "List group direct message channels",
+            "List all group direct message channels",
+            list_group_direct_message_channels_metadata,
+        ),
+        (
+            "List individual direct message channels",
+            "List all individual direct message channels",
+            list_direct_message_channels_metadata,
+        ),
+        (
+            "List direct message channels",
+            "List all direct message channels",
+            list_direct_message_channels_metadata,
+        ),
+        (
+            "List public and private channels",
+            "List public and private channels I am a member of",
+            list_public_channels_metadata,
+            list_private_channels_metadata,
+        ),
+        (
+            "List public channels and direct message conversations",
+            "List public channels and direct message conversations I am a member of",
+            list_public_channels_metadata,
+            list_direct_message_channels_metadata,
+        ),
+    ]
+
+    for name, user_message, *expect_called_tool_functions in cases:
+        suite.add_case(
+            name=name,
+            user_message=user_message,
+            expected_tool_calls=[
+                ExpectedToolCall(
+                    func=tool_function,
+                    args={},
+                )
+                for tool_function in expect_called_tool_functions
+            ],
+        )
+
+    return suite
+
+
+@tool_eval()
+def get_conversations_metadata_eval_suite() -> EvalSuite:
+    """Create an evaluation suite for tools getting conversations metadata."""
+    suite = EvalSuite(
+        name="Slack Tools Evaluation",
+        system_message="You are an AI assistant that can interact with Slack to send messages and get information from conversations, users, etc.",
+        catalog=catalog,
+        rubric=rubric,
+    )
+
+    suite.add_case(
+        name="Get conversation metadata by name",
+        user_message="Get the metadata of the #general channel",
+        expected_tool_calls=[
+            ExpectedToolCall(
+                func=get_conversation_metadata_by_name,
+                args={
+                    "conversation_name": "general",
+                },
+            ),
+        ],
+        critics=[
+            BinaryCritic(critic_field="conversation_name", weight=1.0),
+        ],
+    )
+
+    suite.add_case(
+        name="Get conversation metadata by id",
+        user_message="Get the metadata of the conversation with id '1234567890'",
+        expected_tool_calls=[
+            ExpectedToolCall(
+                func=get_conversation_metadata_by_id,
+                args={
+                    "conversation_id": "1234567890",
```

Comment:
make this harder. these numbers being sequential makes this too easy and unrealistic.